help/en/package/jmri/jmrit/beantable/SensorTable.shtml

```diff
@@ -88,43 +93,106 @@
+          open.  You must use one of the "Store to file" options in the "File" menu to save this new information
+          in a file to be used after JMRI is closed.
+        </dd>
+
+        <dt><strong>Query</strong></dt>
+
+        <dd>
+          Pushing the "Query" button will query the hardware "known state" and put that in the 
+          Sensor state column.         
+        </dd>
+
+        <dt><strong>Forget</strong></dt>
+
+        <dd>
+          Pushing the "Forget" button will reset the Sensor state to "Unknown."
+        </dd>
+
+
+
+      </dl>  
+
       
-      <h2>Sensor Table Controls</h2>
-      <p>Along the bottom of the Sensor Table pane are buttons and checkboxes
-      that control its operation.</p>
+      <h3>Sensor Table Display Controls</h3>
+      <p>Along the bottom of the Sensor Table pane are buttons and checkboxes that control the display of
+      the indicated columns.  Note that the display of these columns can also be changed by right-clicking 
+      on the column headers and selecting or deselecting columns.
+      </p>
 
       <dl>
-        <dt>Add...</dt>
+        <dt><strong>Add...</strong></dt>
 
-        <dd>Push this button to add a new Sensor. For more
-        information see <a href=
-        "SensorAddEdit.shtml">Adding/Editing Sensors</a>.</dd>
+        <dd>A window to add a new Sensor will open when you press this button. For more information see 
+        <a href="SensorAddEdit.shtml">Adding/Editing Sensors</a>.</dd>
 
-        <dt>Show Debounce Delay Information</dt>
+        <dt><strong>Show Debounce Delay Information</strong></dt>
 
         <dd>When checked, the optional <a href=
         "#Debounce">Delay</a> columns are shown.</dd>
 
-        <dt>Show Pull-Up/-Down Information</dt>
-
-        <dd>When checked, the optional <a href=
-        "SensorAddEdit.shtml#pullup">Pull-Up/-Down</a> columns are shown.</dd>
+        <dt><strong>Show Pull-Up/Down Information</strong></dt>
 
-        <dt>Show State Query actions</dt>
+        <dd>When checked, the optional <a href="SensorAddEdit.shtml#pullup">Pull-Up/Down</a> columns are shown.</dd>
 
-        <dd>When checked, an optional Sensor State "Query" and "Forget"
-        button is shown for each Sensor.</dd>
+        <dt><strong>Show State Query actions</strong></dt>
 
-        <dt>Defaults menu</dt>
+        <dd>When checked, an optional Sensor State "Query" and "Forget" button is shown for each Sensor.</dd>
+        
+       </dl>
 
-        <dd><p>The "<a href="#Debounce">Global Debounce Delays...</a>" menu
-        lets item you configure default values for both delays.</p>
-        <p>The "Initial Sensor State" menu item lets you choose what state
-        Internal Sensors will display after startup.</p>
-        </dd>
+       <p>The word <strong>Defaults</Strong> appears on the window's command line.  Clicking it brings up two options:       
```

Comment:
I'm not sure what "command line" means here.  Isn't it just that there's a Defaults menu on th window?